src/Redemption.js

```diff
@@ -94,25 +97,28 @@ export default class Redemption {
           "SignatureSubmitted",
           { digest: redemptionDigest }
         )
-        const { r, s, recoveryID } = signatureEvent.args
+        const { r, s, recoveryID } = signatureEvent.returnValues
```

Comment:
Yeah, they are? Confused as to what you're getting at.

Reply:
Sorry, I asked this here for some reason, but it's because [below](https://github.com/keep-network/tbtc.js/pull/10/files#diff-79aec4708eabeae8da70e4a3ba142f1cR118-R119) you `toString` both of them.

Reply:
Yeah. I think single byte256 words are converted into BN instances by the ABI decoder? Not sure, to be perfectly honest.